### PR TITLE
Copyright

### PR DESCRIPTION
**Pull request by NurkaAmre**
## Context

[link to Jira card](https://ywgc.atlassian.net/jira/software/projects/YWGC/boards/1?selectedIssue=YWGC-33)

## Work done

- [ ] Add website copyright and credits

## Manual testing instructions

- [ ]  Resize the screen to a smaller size to simulate a mobile device.
- [ ]  Check if the website is responsive and adjusts properly to the smaller screen size.

## Gotchas/What I learned

- [ ] Copyright is something important

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,11 +18,8 @@ export default function App () {
       </BrowserRouter>
 
     <Services />
-
+    <Footer />
```
Comment (novellac) [inline src/index.js:21]: issue: This code will place the Footer above the service map area, which I don't think is how the design looks.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div className="bg-green-700 text-white text-center py-14 text-base md:text-xl px-6">
+      <h1>Copyright © 2023 Yorkshire Woods Ground Crew. All rights reserved.</h1>
```
Comment (novellac) [inline src/components/Footer.jsx:6]: issue: Generally, there should only be one h1 per page.
issue: We have a BaseHeader component, so we can use it here.

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <div className='py-8'>
+        <p>Design: <a className='underline' href="https://www.linkedin.com/in/amanda-karczewski/">Amanda Karczewski</a></p>
+        <p>Development: <a className='underline' href="https://github.com/jaynaxissa">Jayna Issariya</a>, <a className='underline' href="https://github.com/NurkaAmre">Nurka Kereikhan</a></p>
+        <p>Product Management: <a className='underline' href="https://github.com/alliequintano">Allie Quintano</a></p>
+      </div>
+      <h1 className='text-center text-white'>Icons from &quot;Lawn Care&quot; collection, created by Berkah Icon, from <a className='underline' href='https://thenounproject.com/browse/collection-icon/lawn-care-163681/?p=1'>Noun Project</a> CC BY 3.0</h1>
```
Comment (novellac) [inline src/components/Footer.jsx:12]: issue: Generally, there should only be one h1 per page.

**Review by novellac — CHANGES_REQUESTED** [review]
issue: Please resolve the merge conflict before I give this a final check.

File: src/index.js (hunk 1 above)
Comment (NurkaAmre) [reply src/index.js:20]: Where I suppose to display the footer?

**Review by novellac — APPROVED** [review]
issue(nb, but if you don't fix it here please make a card for it): When the content doesn't take up the whole screen, there appears a white space at the bottom of the page. This happens especially on large screens.
Screenshot 2023-07-11 at 6 24 17 PM

issue(nb, but please keep in mind for the future): Please update the testing instructions in the PR so reviewsers and QA know what to test.

**Pull request merged**